packages/ncids-js/src/components/nci-header/utils/mega-menu/focus-trap.ts

```diff
@@ -0,0 +1,97 @@
+/**
```

Comment:
search and mobile need this - can it just be in utils?

Reply:
you betcha